files/galaxy/tpv/tool_defaults.yml

```diff
@@ -6,7 +7,8 @@ tools:
     cores: 1
     mem: cores * 3.8
     gpus: 0
-    env: {}
+    env:
+      GALAXY_MEMORY_MB: "{mem * 1024}"
```

Comment:
```suggestion
      GALAXY_MEMORY_MB: "{mem * 1024}" # set 5/2023 might be moved to runner or tool wrappers, related to Galaxy issue 15952
```

Reply:
Alternatively, making this part of the commit message may be equally useful.